### ISSUE TITLE
Domain activation is Failed - Change from S3 to amplify. problem with domain

### ISSUE THREAD

**Issue by SergioRosello** [aws-amplify/amplify-console#763]
**App Id**: dsq5lwrk60v1g
**Region**: eu-west-2
**Step**: Domain activation
**Status**: Failed

**Issue/question**
I had a S3 bucket with a cloudfront to host a website, but now I changed to amplify.
I want to use the same domain, so I removed the domain from cloudfront, but this error mesasge keeps appearing.
Is there anything else I have to do in order to remove the previous connection with the domain? 
I have waited > 24h, as stated in one answer regarding a similar issue.

**Error message**
If there is an error message, please include it here. 
One or more of the CNAMEs you provided are already associated with a different resource.

**Additional information**
Please add any other relevant information. Please feel free to include screenshots.

**Comment by swaminator** [comment]
@SergioRosello could you try deleting and re-adding the domain?

**Comment by SergioRosello** [comment]
Is this what you are asking me to do?
![image](https://user-images.githubusercontent.com/6085350/84748938-9ad9d880-afb9-11ea-992f-4629c07ce218.png)

I feel this is unnecessary, as I could lose the domain, and domain registration fee, which then I would have to re-purchase.
There has to be a alternative solution to this problem.

**Comment by OzzieOrca** [comment]
I might have struggled with the same issue. Does https://github.com/aws-amplify/amplify-console/issues/512#issuecomment-639892355 help?

**Comment by SergioRosello** [comment]
Thanks for the comment.
I went as far as deleting the cloudfront distribution.
![image](https://user-images.githubusercontent.com/6085350/85047034-9536eb00-b191-11ea-85d3-881c081fce68.png)

I supposed it wasn't necessary, as AWS amplify would handle the rest.
Maybe I'm completely wrong.
Now, my problem has transformed, from the mentioned above, to the following:
![image](https://user-images.githubusercontent.com/6085350/85047345-f52d9180-b191-11ea-9229-18725b7b5035.png)

I have verified it exists, but nothing happens
